INSTALL.md

```diff
@@ -164,6 +164,18 @@ Then you can try one of the
 [examples](https://github.com/kubernetes/client-go/tree/v2.0.0/examples/) from
 the 2.0.0 release.
 
+For client-go 3.0.0 must be defined `k8s.io/apimachinery` override in Gopkg.toml
+
+```
+[[constraint]]
+  name = "k8s.io/client-go"
+  version = "3.0.0"
+
+[[override]]
```

Comment:
Could you try that again? I tried `constraint` in my project and it worked.

My Gopkg.toml:
```
[[constraint]]
  name = "k8s.io/apimachinery"
  branch = "release-1.7"

[[constraint]]
  name = "k8s.io/client-go"
  version = "4.0.0"
```

I thought the constraint will be effective unless there is a conflicting contraint in the dependency's Gopkg.toml. Because client-go doesn't have a `Gopkg.toml`, so the contraint in the user's project will be effective.